script/example-data.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+
+require 'csv'
```

Comment:
do we use csv here?

Reply:
Good catch, that's left over from my first draft where I was using CSV.